roles/ara_api/tasks/main.yaml

```diff
@@ -34,12 +34,17 @@
 
 - name: Include configuration of the database engine
   include_tasks: "database_engine/{{ ara_api_database_engine }}.yaml"
+  when: ara_api_install_method != 'podman'
 
 - name: Include installation of the WSGI backend server
   include_tasks: "wsgi_server/{{ ara_api_wsgi_server }}.yaml"
-  when: ara_api_wsgi_server is not none
+  when:
+    - ara_api_wsgi_server is not none
+    - ara_api_install_method != 'podman'
 
 - name: Include installation of the frontend server
   include_role:
     name: "ara_frontend_{{ ara_api_frontend_server }}"
-  when: ara_api_frontend_server is not none
+  when:
+    - ara_api_frontend_server is not none
+    - ara_api_install_method != 'podman'
```

Comment:
An nginx on the host doing reverse proxy and SSL termination on top of a container is a very valid use case.

I think the vhost is even largely the same if not the same ? For example, https://github.com/ansible-community/ara-collection/blob/master/roles/ara_frontend_nginx/templates/ara-api-ssl.conf.j2, if ``ara_api_wsgi_bind`` is the address where the container is running (instead of gunicorn), it should work.

Reply:
Indeed it is, in my current setup I have Apache in front of (I do Kerberos authentication). I'll look into that next time I have some time :)